backend/dal/Helpers/Extensions/ParcelExtensions.cs

```diff
@@ -84,13 +84,39 @@ public static IQueryable<Entity.Parcel> GenerateQuery(this PimsContext context,
             {
                 var pidValue = filter.PID.Replace("-", "").Trim();
                 if (Int32.TryParse(pidValue, out int pid))
-                    query = query.Where(p => p.PID == pid || p.PIN == pid);
+                    query = query.Include(p => p.Buildings)
```

Comment:
This includes the building data if we are doing a pid/pin search, since a pid/pin search is what the FE uses to find a parcel match for map clicks and pid/pin searches.

Alternative would be to make one request to get the id of the pid/pin match, and another followup to get the details.

Reply:
We'll need to do a hot-fix to prod for this.